msteams-platform/messaging-extensions/high-quality-message-extension.md

```diff
@@ -767,24 +767,27 @@ For a plugin to be validated, invoked, and work seamlessly, ensure that it meets
 
 ## Message extensions Plugins in Copilot for Microsoft 365
 
+> [!IMPORTANT]
+> Message extensions Plugins in Copilot for Microsoft 365 are in limited private preview. More details to be published after a public preview is announced.
+
 Copilot extensions customize and extend the Copilot for Microsoft 365 experience by bringing more skills and knowledge to Copilot for a personalized user experience. By using plugins, which are a subset of Copilot extensions, users can integrate additional capabilities into Copilot by interacting with third-party applications, whether for retrieving or modifying information within those apps. For instance, message extension plugins facilitate searching for data in other applications so that Copilot can present it upon request when the plugin is activated.
```

Comment:
Two observations:
1. This section is about Copilot in M365 apps, specifically in Word and PowerPoint. Saying M365 Copilot is misleading, as MEs are GAd and available to customers as plugins in M365 Copilot today. But in private preview in Copilot in Word and PowerPoint. Can we add this clarification to the title of the section and also to the note, please? Suggestions:
For the title: Message extensions plugins in Copilot for M365 applications
For the note: Message extensions plugins in Copilot for M365 applications are in limited private preview for Word and PowerPoint. More details to be published after a public preview is announced.
2. In the title we are capitalizing the word "Plugins", can we use small caps there, please?

Reply:
@isvargasmsft Comments are incorporated in the PR.